<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormsWdg</class>
 <widget class="QWidget" name="FormsWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Questionnaires</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QToolButton" name="tbDel">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>70</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Delete a questionnaire</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="fingram.qrc">
        <normaloff>:/resources/delete.png</normaloff>:/resources/delete.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>4</y>
        <width>52</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#00007f;&quot;&gt;Forms&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QToolButton" name="tbEdit">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>110</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Edit a questionnaire</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="fingram.qrc">
        <normaloff>:/resources/edit.png</normaloff>:/resources/edit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="tbAdd">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>30</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Add a questionnaire</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="fingram.qrc">
        <normaloff>:/resources/add.png</normaloff>:/resources/add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="tbPrint">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>160</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Edit a questionnaire</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="fingram.qrc">
        <normaloff>:/resources/print.png</normaloff>:/resources/print.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tvForm">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tvForm</tabstop>
  <tabstop>tbAdd</tabstop>
  <tabstop>tbDel</tabstop>
  <tabstop>tbEdit</tabstop>
 </tabstops>
 <resources>
  <include location="fingram.qrc"/>
 </resources>
 <connections/>
</ui>
